cli/generate_operator_keys.go

```diff
@@ -15,6 +19,25 @@ var generateOperatorKeysCmd = &cobra.Command{
 	Use:   "generate-operator-keys",
 	Short: "generates ssv operator keys",
 	Run: func(cmd *cobra.Command, args []string) {
+		passwordFilePath, _ := cmd.Flags().GetString("password-file")
+
+		// Resolve to absolute path
+		absPath, err := filepath.Abs(passwordFilePath)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		// Now read the file
+		// #nosec G304
+		passwordBytes, err := os.ReadFile(absPath)
+		if err != nil {
+			log.Fatal(err)
+		}
+		if err != nil {
+			log.Fatal(err)
+		}
+		encryptionPassword := string(passwordBytes)
```

Comment:
@moshe-blox If a user leaves a new line in the password file, should it be a part of the encryption password?

Reply:
IMO we don't have to do as full proof, if we trim the file and the user intended to have a space inside?

Reply:
if @moshe-blox agrees with it I don't care to trim the string though.

Reply:
@moshe-blox @y0sher alright, let's not trim it then